hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/rsql/RSQLUtilityTest.java

```diff
@@ -99,6 +100,43 @@ TenantConfigurationManagementHolder tenantConfigurationManagementHolder() {
     private static final TenantConfigurationValue<String> TEST_POLLING_OVERDUE_TIME_INTERVAL = TenantConfigurationValue
             .<String> builder().value("00:07:37").build();
 
+    @Test(expected = RSQLParameterUnsupportedFieldException.class)
```

Comment:
I wouldn't define the expected exception like that: first of all it is not consistent with the rest of the test class and secondly it is deprecated with Junit 5, so it would make future migration more involved. I would suggest either following the used pattern of try/catch/fail or use the AssertJ with its assertThatExceptionOfType(exceptionType).isThrownBy(throwingCallable)

Reply:
I decided to go ahead with the functional style using AssertJ since it is less code and hence more readable.